lib/saluki-context/src/resolver.rs

```diff
@@ -564,6 +567,81 @@ async fn drive_telemetry(context_cache: Arc<ContextCache>, interner: GenericMapI
     }
 }
 
+mod private {
+    pub trait Sealed {}
```

Comment:
TIL: Sealed Traits (though searching the codebase I see we use this concept elsewhere too :hmmnod: )